export_weekly_advert_contacts.py

```diff
@@ -80,14 +80,16 @@
     for listening_group_csv_url in pipeline_configuration.listening_group_csv_urls:
         listening_group_csvs_names.append(listening_group_csv_url.split("/")[-1])
 
-    for listening_group_csv in listening_group_csvs_names:
-        with open(f"{listening_group_data_dir}/{listening_group_csv}", "r", encoding='utf-8-sig') as f:
-            data = list(csv.DictReader(f))
-            log.info(
-                f"Loaded {len(data)} listening group participants from {listening_group_data_dir}/{listening_group_csv}")
-            for row in data:
-                uuids.add(row['avf-phone-uuid'])
-                listening_group_participants +=1
+    for k in pipeline_configuration.listening_group_csv_urls.keys():
```

Comment:
Can you replace 'k' with something descriptive?

Reply:
updated to listening_group_type thanks.